### DISCUSSION TITLE
(Completed) [wiki] Python 3.11 standard library `tomllib`

### DISCUSSION THREAD

**Discussion by YDX-2147483647** [toml-lang/toml#937] [General]
A version of Tomli, the tomllib module, was added to the _standard library_ in Python 3.11 via [PEP 680](https://www.python.org/dev/peps/pep-0680/). (→ [Tomli's README](https://github.com/hukkin/tomli/blob/5646e6923d895725aad7ecfa32be19861812d1fc/README.md?plain=1#L35-L37))

I think we should add it to [the list in wiki](https://github.com/toml-lang/toml/wiki#v100-compliant).

1. Tomli is also in the list. Should we list them together?
2. [The standard library does **NOT** support _writing_ TOML.](https://docs.python.org/3.11/library/tomllib.html) Does it fit the criteria? (Edit: Tomli does not support either.)

**Comment by ChristianSi** [comment]
Yes, I'd say that since tomllib *is* tomli, they should be listed together. And I'd move them to the top of the Python section, since that TOML parser is now as official as you can get in the Python world.

Neither of them can write TOML documents (as you observe). That job is left to tomli-w, which is already listed separately in the "Encoders" section. And that's fine.
